src/client.cpp

```diff
@@ -908,6 +905,56 @@ void CClient::Stop()
     SignalLevelMeter.Reset();
 }
 
+/// @method
```

Comment:
This should also document what signals are emitted.  The signals should be documented as to how they're intended to be used, too.

Reply:
@ emit does this. But Ofc, we can rewrite what we want to document.